### ISSUE TITLE
Media Queries for other sizes other than Large

### ISSUE THREAD

**Issue by cubadomingo** [cubadomingo/words_and_moves#30]
So the site looks pretty good in a desktop view, but it's bit buggy in other sizes. Media queries should be done for:

- xs

- s

-m

I usually like to separate the media queries in different partials and then import them into the main  application file. The views that need changing are:

- landing page
- sign in view
- sign up view
- forgot password view

**Comment by tmobaird** [comment]
I don't much about media queries, but I believe bootstrap has media queries built in. I believe all grid column sizes scale up. Ex. medium will scale on all screens medium - lg/xlarge. IF this is the case, I believe it's best to go with a small col size like xs/sm/md. These should scale properly for us.

I might be misunderstanding the problem though. I've tried the pages on smaller screens and they appear to be working nicely.

**Comment by cubadomingo** [comment]
Okay so looking back at it, you're right I don't think it's a media query issue per say. It does look pretty good in mobile but here's the bugs I was referring to.

![img_1816](https://cloud.githubusercontent.com/assets/15992549/17248094/08b46a9e-5566-11e6-9af9-5609b4fbc029.PNG)
![img_1817](https://cloud.githubusercontent.com/assets/15992549/17248099/0cd44cb6-5566-11e6-86c9-9e13d299ad78.PNG)

So actually I think this is a CSS issue. It'll look into tackling this problem today.

**Comment by tmobaird** [comment]
Yep I think you're right. There might be a bootstrap class that fixes an element to the bottom of the screen
